### ISSUE TITLE
Incorrect product key variable used in makefile for comaparison

### ISSUE THREAD

**Issue by 10Peppermill** [canonical/packer-maas#255]
On line 42 of packer-maas/windows/makefile a comparison between the variable `$(PKEY1)` and nothing is used to determine if the autounattended needs to be modified to include a product key. Since the comparison is of the undefined variable PKEY1 the autounattended file never gets updated with a product key. Replacing PKEY1 with PKEY fixes the issue.

https://github.com/canonical/packer-maas/blob/87b80962783979958640c88654d8b7615bc0aa6a/windows/Makefile#L42

**TLDR:** someone fatfingerd a 1 onto the end of a variable name 😆

**Comment by alexsander-souza** [comment]
Hi @10Peppermill, thank you for looking into this.
